test/verify/check-machines

```diff
@@ -516,5 +528,436 @@ class TestMachines(MachineCase):
+            time.sleep(1)  # wait for virt-install to finish
+            runner.deleteVm(dialog)
+            checkEnvIsEmpty()
+
+        def installTest(dialog):
+            runner.tryInstall(dialog)
+            time.sleep(1)  # wait for virt-install to finish
+            runner.deleteVm(dialog)
+            checkEnvIsEmpty()
+
+        checkEnvIsEmpty()
+
+        # test just the DIALOG CREATION and cancel
+        print("    *\n    * validation errors and ui info/warn messages expected:\n    * ")
+        cancelDialogTest(TestMachines.VmDialog(b, "subVmTestCreate1", is_filesystem_location=False,
+                                               location=runner.CENTOS_URL,
+                                               memory_size=12654, memory_size_unit='GiB',
+                                               storage_size=0, storage_size_unit='MiB',
+                                               start_vm=False))
+
+        cancelDialogTest(TestMachines.VmDialog(b, "subVmTestCreate2", is_filesystem_location=True,
+                                               location=runner.CENTOS_MOCKUP_ISO_PATH,
+                                               memory_size=1, memory_size_unit='MiB',
+                                               storage_size=12500, storage_size_unit='GiB',
+                                               os_vendor=runner.CENTOS_VENDOR,
+                                               os_name=runner.CENTOS_OS,
+                                               start_vm=True))
+
+        # try to CREATE WITH DIALOG ERROR
+
+        # name
+        checkDialogErrorTest(TestMachines.VmDialog(b, ""), "Name")
+
+        # location
+        checkDialogErrorTest(TestMachines.VmDialog(b, "subVmTestCreate4", is_filesystem_location=False,
+                                                   location="invalid/url"), "Source")
+
+        # memory
+        checkDialogErrorTest(TestMachines.VmDialog(b, "subVmTestCreate5", location=runner.CENTOS_MOCKUP_ISO_PATH,
+                                                   memory_size=1000000000, memory_size_unit='GiB',
+                                                   storage_size=100, storage_size_unit='MiB',
+                                                   start_vm=True), "memory")
+
+        # disk
+        checkDialogErrorTest(TestMachines.VmDialog(b, "subVmTestCreate6", location=runner.CENTOS_MOCKUP_ISO_PATH,
+                                                   storage_size=1000000000, storage_size_unit='GiB',
+                                                   start_vm=True), "capacity")
+
+        # start vm
+        checkDialogErrorTest(TestMachines.VmDialog(b, "subVmTestCreate7", start_vm=True),
+                             "An install method must be specified")
+
+        # try to CREATE few machines
+        createTest(TestMachines.VmDialog(b, "subVmTestCreate8", storage_size=1))
+
+        createTest(TestMachines.VmDialog(b, "subVmTestCreate9", is_filesystem_location=False,
```

Comment:
Is it actually downloading from the CENTOS_URL?
There might be no access to external network within test execution.

Reply:
no it is not because start_vm=False